scripts/install_server_from_src.sh

```diff
@@ -15,67 +14,133 @@ set -e
 : ${DOWNLOAD_CLIENT_TEMPLATES:=true}
 : ${PYTHON_INSTALL:=true}
 : ${DOCKER:=false}
+: ${INSTALL_BIN:="install"}
+: ${INSTALL_OPTS:="-p -m644"}
+: ${SRC_DIR:="/grr"}
+
+function header()
+{
+  echo ""
+  echo "##########################################################################################"
+  echo "     ${*}";
+  echo "##########################################################################################"
+}
+
+OPTIND=1
+while getopts "h?ls:" opt; do
```

Comment:
install_script_ubuntu.sh does the same thing for UPGRADE.  Command line definitely makes it easier to use with Vagrant and as a standalone script.  I can move the rest of the options to command line and break things out a little better to support non-ubuntu installs as well.